AlphaWallet/Tokens/ViewModels/TokenViewControllerViewModel.swift

```diff
@@ -3,13 +3,16 @@
 import Foundation
 import UIKit
 import TrustKeystore
+import web3swift
+import Alamofire
+import PromiseKit
 
-struct TokenViewControllerViewModel {
+class TokenViewControllerViewModel {
```

Comment:
Can you restore this to `struct` and the `recentTransactions` to `let` if you don't need them to be mutable anymore? I am a bit wary because there is 1 case in our codebase where there is a bug triggered simply by changing the view model from class -> struct, or vice versa. I don't remember which one, but I added a comment to that class/struct.

So just to be safe :)

Reply:
Ah. It's: `TokensViewModel`